backend/api/annotations/resources.py

```diff
@@ -1,153 +1,169 @@
-from flask_restful import Resource, current_app, request
-from schematics.exceptions import DataError
+from databases import Database
+from fastapi import APIRouter, Depends, Request
+from loguru import logger
+from starlette.authentication import requires
+
+from backend.db import get_db
+from backend.models.dtos.user_dto import AuthUserDTO
 from backend.models.postgis.task import Task
 from backend.models.postgis.task_annotation import TaskAnnotation
 from backend.services.project_service import ProjectService
 from backend.services.task_annotations_service import TaskAnnotationsService
-from backend.services.application_service import ApplicationService
+from backend.services.users.authentication_service import login_required
 
+router = APIRouter(
+    prefix="/projects",
+    tags=["projects"],
+    dependencies=[Depends(get_db)],
```

Comment:
I'm not sure this is needed here. Some endpoints may not need a db connection and this would create one anyway. I'm not sure how you would access the database connection variable like this anyway